src/server/pfs/pfs.go

```diff
@@ -146,6 +139,20 @@ type ErrDropWithChildren struct {
 	Commit *pfs.Commit
 }
 
+// ErrInvalidBranchStructure represents an error where more than one branch
```

Comment:
Maybe this should be `ErrInvalidDAGStructure` or `ErrInvalidProvenanceStructure`?